packages/test/test-service-load/testConfig.json

```diff
@@ -36,7 +36,11 @@
 						"Fluid.Container.summarizeProtocolTree2": [true, false]
 					}
 				}
-			}
+			},
+			"useRandomContent": true,
```

Comment:
should these be grouped, its not clear that they are related when flatly listed on the config